server/cmwell-fts/src/main/scala/cmwell/fts/FTSService.scala

```diff
@@ -982,10 +990,17 @@ class FTSService(config: Config) extends NsSplitter{
     def filterToBuilder(filter:AggregationFilter):AggregationBuilder = {
 
       implicit def fieldValueToValue(fieldValue: Field): String = fieldValue.operator match {
-        case AnalyzedField =>  reverseNsTypedField(fieldValue.value)
+        case AnalyzedField =>
+          if(isStrSystemField(fieldValue.value))
+            throw new IllegalArgumentException("aggregations failure due to text system field")
+          val fType = fieldType(fieldValue.value)
+          if(fType eq StringType) {
+            throw new IllegalArgumentException("aggregations failure due to fielddata disabled")
+          }
+          else reverseNsTypedField(fieldValue.value)
         case NonAnalyzedField=>
           val fType = fieldType(fieldValue.value)
```

Comment:
String system fields cannot be aggregated.

Reply:
done